### ISSUE TITLE
Allow import from absolute URLs (or passthrough import mode)

### ISSUE THREAD

**Issue by AshleyScirra** [google/closure-compiler#3708]
Closure Compiler rejects import statements from an absolute URL. This should be easy to support if it just passes them through verbatim. Instead any presence of such a line causes the entire script to fail to compile.

Example in.js:

```js
import test from "https://cdn.example.com/test.js";
```

Command line:
`java -jar ./closure-compiler.jar --js in.js --js_output_file out.js --compilation_level SIMPLE --language_in ECMASCRIPT_2020 --language_out ECMASCRIPT_2020`

Observed result:
```
in.js:2:0: ERROR - [JSC_INVALID_MODULE_PATH] Invalid module path "https://cdn.example.com/test.js" for resolution mode "BROWSER"
  2| import test from "https://cdn.example.com/test.js";
     ^

1 error(s), 0 warning(s)
```

Expected result:

Output file to contain the same import statement.

Similar to issue #3707, if there is a passthrough module resolution mode, we can just set it to output import statements verbatim and this case will work. This makes it much easier to use Closure Compiler as a general minifier tool, since otherwise the presence of module features causes the whole compilation to fail.

**Comment by concavelenz** [comment]
Are you hoping for a "file by file" compilation here?  Or are you hoping the resulting bundle would preserve the import?

For one off dependencies, you can use the output wrapper to wrap the compiler's output to make it a module.

**Comment by AshleyScirra** [comment]
Yes, I think this is basically a "file by file" case - we're basically using Closure Compiler as just a minifier, because it is probably the most sophisticated minifier anyone's built.

Since we're just minifying code, we don't care about changing imports at all, so it's fine for Closure Compiler to just pass through imports unmodified.

I'm not sure what you mean about using an output wrapper - how does that help in this case? We need this to work in general though, not for one-off cases.
